Packs/Armis/Integrations/Armis/Armis.py

```diff
@@ -257,24 +291,49 @@ def free_string_search_devices(self, aql_string: str, order_by: str = None, max_
         return self.search_by_aql_string(f"in:devices {aql_string}", order_by=order_by, max_results=max_results)
 
 
-def test_module(client: Client):
+def test_module(client: Client, params: dict):
     """
     Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
     This test works by using a Client instance to create a temporary access token using the provided secret key,
     thereby testing both the connection to the server and the validity of the secret key
     Args:
         client: Armis client
+        params: A dictionary containing the parameters provided by the user.
     Returns:
         'ok' if test passed, anything else will fail the test.
     """
 
     try:
-        client._get_token(force_new=True)
+        if params.get('isFetch', False):

```

Comment:
```suggestion
        if argToBoolean(params.get('isFetch', False)):
```
Since bool("false") == True
Please use the [argToBoolean](https://xsoar.pan.dev/docs/reference/api/common-server-python#argtoboolean) function.

Reply:
Hi @MosheEichler, we have updated the code as per your suggestion.